src/codec/decodeMeasurement-delta-protobuf.js

```diff
@@ -14,5 +14,18 @@ module.exports = function(buffer){
     protoDecoded.forEach(function(measurement){
         measurement.signal_strengths = deltaDecode(measurement.signal_strengths);
     });
-    return protoDecoded;
+    return protoDecoded.map(function (measurement) {
+
+        var devices = measurement.signal_strengths.map(function (signal_strength, index) {
+            return {
+                signal_strength: signal_strength,
+                ID: (measurement.IDs && index < measurement.IDs.length) ? measurement.IDs[index] : undefined
```

Comment:
Good point. Add a comment to explain this. And maybe a TODO to remove the check to be done when all sensors are updated. 

We shouldn't be keeping cruft from the past too long if the past moves past the present... I mean... you understand what I mean.